scripts/verify-gopkg.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+TOML=$( grep -P '( |^)name = ".*"' Gopkg.toml | tr -d ' ' | sort )
+LOCK=$( grep -P '( |^)name = ".*"' Gopkg.lock | tr -d ' ' | sort )
+
+# NOTE(ericchiang): For some reason you need this "export" here for bash reasons
+# I don't understand.
+export DIFF=$( diff <(echo "$TOML") <(echo "$LOCK") )
```

Comment:
should be:

```
DIFF=$( diff <(echo "$TOML") <(echo "$LOCK") || true )
```

`diff` will return an error code if there is a difference causing the script to exit.

Reply:
it worked!